pallets/ocex/src/lib.rs

```diff
@@ -116,13 +90,13 @@ decl_error! {
 decl_storage! {
     trait Store for Module<T: Config> as OCEX {
         LastAccount get(fn key) config(): T::AccountId;
-        pub MainAccounts get(fn get_main_accounts): map hasher(blake2_128_concat) T::AccountId => LinkedAccount<T>;
+        pub MainAccounts get(fn get_main_accounts): map hasher(blake2_128_concat) T::AccountId => LinkedAccount;
     }
     add_extra_genesis {
         config(genesis_account): T::AccountId;
         build( |config: &GenesisConfig<T>| {
 
-            let linked_account_object = LinkedAccount::from(config.genesis_account.clone(), config.genesis_account.clone());
+            let linked_account_object = LinkedAccount::from(AccountId::new(config.genesis_account.clone().encode().as_slice().try_into().unwrap()), AccountId::new(config.genesis_account.clone().encode().as_slice().try_into().unwrap()));
```

Comment:
This is not right, the sizes of do not match, therefore it fails to make the array as it expects `[u8; 32]`

Reply:
Resolved

Reply:
Never use unwrap in Substrate Runtime.

@zktony Please provide a better solution for creating LinkedAccount struct.